setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
maybe it was some other pyqt problem - I also have encountered one I can't solve, see #1.